packages/tasks/src/local-apps.ts

```diff
@@ -47,10 +47,11 @@ function isGgufModel(model: ModelData) {
 }
 
 const snippetLlamacpp = (model: ModelData): string => {
-	return `./main \
-	--hf-repo ${model.id} \
-	-m file.gguf \
-	-p "I believe the meaning of life is " -n 128`;
+	return `./main \\
+	--hf-repo "${model.id}" \\
+	-m file.gguf \\
+	-p "I believe the meaning of life is " \\
+	-n 128`;
```

Comment:
nit:

```suggestion
	return [".main", `--hf-repo "${model.id}"`, "-m file.gguf",  '-p "I believe the meaning of life is "', "-n 128"].join(' \\\n');
```

100% optional

Reply:
keeping like it is for consistency, ie 
https://github.com/huggingface/huggingface.js/blob/d1c0cb8604db4f504fa33b233fc12b9a501abc17/packages/tasks/src/snippets/curl.ts#L5-L11